index.js

```diff
@@ -108,6 +108,22 @@ const timeout = (promise, ms) => Promise.race([
 	})()
 ]);
 
+const isAbsoluteUrl = str => /^[a-z]+:\/\//.test(str);
+
+const createRequestUrl = (prefixUrl, input) => {
+	if (!prefixUrl) {
+		return input;
+	}
+
+	if (prefixUrl && input.startsWith('/')) {
```

Comment:
I somewhat feel that this exception is a bit arbitrary and tied to a very specific use case of passing a URL as query parameter. I also found this in the tests as something like

```
key.get('https://bar.com', { prefix: 'https://foo.com?redirect=' })
```

If a user decides to use `ky` in such a way, I don't see why the library should handle exceptions for this. I'd lean on the other end and allow users to do any of the following:

```
key.get('users', { prefix: 'https://foo.com/api' })
key.get('/users', { prefix: 'https://foo.com/api' })
key.get('/users', { prefix: 'https://foo.com/api/' })
```

These all seem like more relevant convenience use cases than preventing users from breaking the above-mentioned query case example.

Reply:
I agree, but many people would be confused.

`/` means root of the tree. But in that case the root is different, it's hard to choose.

Reply:
Cool.